backend/supabase/posts/triggers.sql

```diff
@@ -29,6 +29,3 @@ or
 update on posts for each row
 execute function post_populate_cols ();
 
-update posts
-set
-  fs_updated_time = fs_updated_time;
```

Comment:
Wasn't there a purpose for this?

Reply:
yeah but Marshall was against having it here incase someone was just blindly running these